.github/workflows/i18n-pull.yml

```diff
@@ -10,55 +10,6 @@ on: workflow_dispatch # manually trigger
 
 jobs:
   i18n-pull:
-    runs-on: ubuntu-20.04
-    env:
-      PYTHON-VERSION: 3.9
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-
-      # setup python
-      - name: Set up Python ${{ env.PYTHON-VERSION }}
-        uses: actions/setup-python@v2
-        with:
-          python-version: ${{ env.PYTHON-VERSION }}
-
-      # install dependencies
-      - name: Install dependencies
-        run: |
-          pip install -e ".[all]"  
-
-      # install transifex client
-      - name: transifex-client
-        run: |
-          pip install transifex-client
-
-      # store token
-      - name: store token
-        run: |
-          touch ~/.transifexrc
-          echo [https://www.transifex.com] >> ~/.transifexrc
-          echo api_hostname=https://api.transifex.com >> ~/.transifexrc
-          echo hostname=https://www.transifex.com >> ~/.transifexrc
-          echo password=${{ secrets.TRANSIFEX_TOKEN }} >> ~/.transifexrc
-          echo username=api  >> ~/.transifexrc
-
-      # pull all lang
-      - name: pull translations
-        run: |
-          tx pull -a -f
-
-      # remove transifex credential file
-      - name: remove transifexrc file
-        uses: JesseTG/rm@v1.0.2
-        with:
-          path: ~/.transifexrc
-
-      - name: Create Pull Request
-        uses: peter-evans/create-pull-request@v3
-        with:
-          commit-message: i18n:pulled translations
-          title: i18n:pulled translations
-          body: i18n:pulled translations
-          branch: 101-translations-pull  # name of branch which PR is created from
-          delete-branch: true  # delete branch once merged,closed
+    uses: inveniosoftware/invenio-i18n/.github/workflows/reusable/i18n-pull.yml
```

Comment:
```suggestion
    uses: inveniosoftware/invenio-i18n/.github/workflows/reusable/i18n-pull.yml@main
```

Reply:
changed to `@master` since we do not have a `main` branch